### PR TITLE
design system core documentation

### PR DESCRIPTION
**Pull request by themonster2015**
## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: libs/design-system-core/README.md [hunk 1]
```diff
@@ -1,7 +1,70 @@
...
+```tsx
+  <Card elevation={1} padding={'p-4'}>
+    <ExampleComponent1>
+    <ExampleComponent2>
+  </Card>
+```
+
+### Button
+
+A button component with rounded borders. Has 2 variants: `primary` for the call-to-action
```
Comment (didd) [inline libs/design-system-core/README.md]: Has three variants :) text as well

File: libs/design-system-core/src/components/Accordion/index.tsx [hunk 2]
```diff
@@ -17,6 +17,40 @@ export type AccordionProps = {
...
+ * </Stack>
+ * );
+ *
+ * <Accordion
+ *  open: true
+ *  accordionId: 'someId'
+ *  titleNode: title
+ *  contentNode: content
+ *  customStyle: 'w-[15%]'
+ * />
```
Comment (josenriagu) [inline libs/design-system-core/src/components/Accordion/index.tsx]:
```suggestion
* <Accordion open={true} accordionId='someId' titleNode={title} contentNode={content} customStyle='w-[15%]' />
```

File: libs/design-system-core/src/components/AppAvatar/index.tsx [hunk 3]
```diff
@@ -13,6 +13,23 @@ export type AppAvatarProps = AvatarProps & {
...
+ * @example
+ * ```tsx
+ * const profileId = 'did:pkh:eip155:5:0x36c703c42dfa2437dc883e2e0884e57404e16493';
+ * const avatar = { src: 'https://placebeard.it/360x360', height: 360, width: 360 };
+ *
+ * <AppAvatar
+ *   appType: IntegrationTypes.APP
+ *   profileId: profileId
+ *   avatar
+ * />
```
Comment (josenriagu) [inline libs/design-system-core/src/components/AppAvatar/index.tsx]:
```suggestion
 * <AppAvatar avatar={avatar} appType={IntegrationTypes.APP} profileId={profileId} />
```

File: libs/design-system-core/src/components/AppAvatar/index.tsx [hunk 4]
```diff
@@ -13,6 +13,23 @@ export type AppAvatarProps = AvatarProps & {
   appType: IntegrationTypes;
 };
 
+/**
+ *  An AppAvatar provides a fast and easy way to create an avatar component.
```
Comment (josenriagu) [inline libs/design-system-core/src/components/AppAvatar/index.tsx]:
```suggestion
 * An AppAvatar provides a fast and easy way to create an avatar component.
```

File: libs/design-system-core/src/components/AppIcon/index.tsx [hunk 5]
```diff
@@ -34,6 +34,25 @@ export type AppIconProps = {
   customStyle?: string;
 };
 
+/**
+ *  An AppIcon component provides a fast and easy way to include a custom app icon
```
Comment (josenriagu) [inline libs/design-system-core/src/components/AppIcon/index.tsx]:
```suggestion
 * An AppIcon component provides a fast and easy way to include a custom app icon
```

**Review by josenriagu — CHANGES_REQUESTED** [review]

**Review by quininez — APPROVED** [review]

**Review by josenriagu — APPROVED** [review]

**Pull request merged**